Left nav enhancement

anmol-algo opened the pull request:
Updated the left nav changes

steveyen commented:
hmm, looks like there's a second dropdown menu that appears when the reader is on an SDK page.  will fix that too with another commit that i'll add to this pull request.

steveyen commented:
hmmm, this is missing the dropdown carets and the changing of the secondary menu (SDK) title changes when the version changes.  Going to close this the PR and fix it, then push up a new PR.

cb-docs-robot committed:
Release sandbox-1

cb-docs-robot committed:
Release sandbox-1

anmol-algo committed:
update tutorial page remove inline html code

anmol-algo committed:
update tutorial page remove inline html code

anmol-algo committed:
remove console logs

anmol-algo committed:
remove console logs

anmol-algo committed:
revert tutorial page changes from this branch.

anmol-algo committed:
revert tutorial page changes from this branch.

cb-docs-robot committed:
Release prod-80

cb-docs-robot committed:
Release prod-80

cb-docs-robot committed:
Release sandbox-2

mojavelinux committed:
update UI to leverage features in Antora 3

- switch back to built-in canonical URL (page.canonicalUrl)
- switch back to built-in latest page (page.latest)
- don't show View Latest button if page does not exist in latest version
- minor formatting

mojavelinux committed:
update UI to leverage features in Antora 3

- switch back to built-in canonical URL (page.canonicalUrl)
- switch back to built-in latest page (page.latest)
- don't show View Latest button if page does not exist in latest version
- minor formatting

mojavelinux committed:
add canonical link to all pages for which page.canonicalUrl is set

anmol-algo committed:
Resolve banner issue

anmol-algo committed:
update top nav bar style

anmol-algo committed:
update top nav bar style

anmol-algo committed:
revert banner code

anmol-algo committed:
revert banner code

anmol-algo committed:
resolved issues

cb-docs-robot committed:
Release sandbox-3

anmol-algo committed:
update docs site header and ui bug fix

anmol-algo committed:
update docs site header and ui bug fix

anmol-algo committed:
update docs site header and ui bug fix

anmol-algo committed:
update hover image for documentation

anmol-algo committed:
update hover image for documentation

amarantha-k committed:
fix linting error

cb-docs-robot committed:
Release sandbox-4

anmol-algo committed:
update left navigation

anmol-algo committed:
update left navigation

anmol-algo committed:
updated the navigation grouping

anmol-algo committed:
updated the navigation grouping

mojavelinux commented:
As far as I can see, the navigation is not currently working. I see blank entries in the navigation rather than the component names. Can you do whatever you have to do to make the unified navigation work in the preview? I'm going to use this as a prototype to develop the client-side navigation, so I'm not concerned with how this is implemented. I just need to see it working.

Can you also rebase this against master so that there's only a single commit rather than 45?

Thanks!

mojavelinux commented:
> Can you do whatever you have to do to make the unified navigation work in the preview?

By that I mean you can modify the preview-src/ui-model.yml. Since I'm going to reimplement it, it's not going to cause a problem in the final version.
